src/groupWith.js

```diff
@@ -8,23 +8,23 @@ var _curry2 = require('./internal/_curry2');
  * @memberOf R
  * @since v0.21.0
  * @category List
- * @sig (a, a -> Boolean) -> [a] -> [[a]]
+ * @sig ((a, a) → Boolean) → [a] → [[a]]
```

Comment:
Somebody showed me this yesterday, not sure if this is what you're looking for:

https://github.com/i-tu/Hasklig